bin/release.sh

```diff
@@ -48,6 +49,15 @@ S3_BASE_URL="s3://net-mozaws-prod-delivery-contrib$S3_ROOT_PATH"
 aws s3 cp devtools-signed.xpi $S3_BASE_URL/devtools.xpi
 ADDON_URL="https://archive.mozilla.org/pub/labs/devtools/master/devtools.xpi"
 
+# Feth the final addon version from install.rdf
+# -E is for using regexp (not only strings) and -o is for printing only the pattern that matches
+VERSION=$(grep -E "em:version=\"([0-9.ab-]+)\"" $SCRIPT_DIR/../install.rdf | grep -oE "([0-9.ab-]+)")
```

Comment:
Yes, it should work. I imagine the best solution would be to have a template instead.